src/snowflake/connector/log_configuration.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+from __future__ import annotations
+
+import logging
+import os
+from datetime import datetime
+
+from snowflake.connector.config_manager import CONFIG_MANAGER
+from snowflake.connector.constants import CONNECTIONS_FILE
+from snowflake.connector.secret_detector import SecretDetector
+
+
+class EasyLoggingConfigPython:
+    def __init__(self):
+        self.log_path = None
+        self.log_level = None
+        self.log_file_name = None
+        self.parse_config_file()
+
+    def parse_config_file(self):
+        CONFIG_MANAGER.read_config()
+        data = CONFIG_MANAGER["connections"]
```

Comment:
I'd like @sfc-gh-mkeller and @sfc-gh-turbaszek to weigh in on where we want to put the logging settings.  A couple of things to consider:

* Should each connection be able to log in a different place?
* We have precedence in how the [CLI sets up logging](https://docs.snowflake.com/en/developer-guide/snowflake-cli-v2/connecting/configure-logging) so we should at least be consistent with that approach.  I'm not saying that connector logging should go in `[cli.logs]` but we want at least the same key names and setting semantics (unless, if we want to change it we should work with that team to keep consistent).

My gut feeling is that we want a top level `[logs]` section in `config.toml` with the same settings and semantics as in the `[cli.logs]` section.  `[cli.logs]` could inherit or override this (eventually) and if we want per-connection logging, they would alos inherit or override.  We also need to update or write any public documentation about the format for `config.toml`.

Maybe a separate 1-pager to define the semantics and syntax would help, with at least Mark, Tomasz, and myself as reviewers?

Reply:
I think we should keep all logs at the same place.
For semantics of config file, I think we should follow the design doc of easy logging and keep it the same as other drivers, otherwise wouldn't it make this project unnecessary? Correct me if I understand your comment wrong

Reply:
> design doc of easy logging

Do you mean: https://docs.google.com/document/d/1zV3E-288HxJKJv5X0ct1lxXcuYGNMcF3--P30xdxJZk/edit#heading=h.5o4hope6ubn0 ?

If so, I'm concerned that we don't have alignment on that yet, and also that SnowCLI has forged a different path, which -- if that's temporary then fine, but if not won't it be confusing that our client side tools and libraries all have different ways to specify log configuration?